docs/workflow/testing/libraries/testing-apple.md

```diff
@@ -16,14 +16,17 @@ Unlike XHarness-Android, XHarness for iOS is able to boot simulators by its own.
 - The following shows how to run tests for a specific library on a simulator
 ```
 cd src/libraries/System.Numerics.Vectors/tests
-~runtime/dotnet.sh build /t:Test /p:TargetOS=iOS /p:TargetArchitecture=x64
+./dotnet.sh build /t:Test /p:TargetOS=iOS /p:TargetArchitecture=x64
 ```
 for devices you need to specify `DevTeamProvisioning`:
 ```
-cd src/libraries/System.Numerics.Vectors/tests
-~runtime/dotnet.sh build /t:Test /p:TargetOS=iOS /p:TargetArchitecture=arm64 /p:DevTeamProvisioning=...
+./dotnet.sh build /t:Test /p:TargetOS=iOS /p:TargetArchitecture=arm64 /p:DevTeamProvisioning=...
 ```
 AppleAppBuilder generates temp Xcode projects you can manually open and resolve provisioning there using native UI and deploy to your devices and start debugging.
 
+### How the tests work
+iOS/tvOS *.app (or *.ipa) is basically a simple [ObjC app](https://github.com/dotnet/runtime/blob/master/src/mono/msbuild/AppleAppBuilder/Templates/main-console.m) that inits the Mono Runtime. This Mono Runtime starts a simple xunit test
+runner called XHarness TestRunner which runs tests for all *.Tests.dll libs in the bundle. There is also XHarness.CLI tool with `mlaunch` inside to deploy `*.app` and `*.ipa` to a target (device or simulator) and listens for logs via sockets.
```

Comment:
To make sure GitHub doesn't render it as italics:
```suggestion
runner called XHarness TestRunner which runs tests for all `*.Tests.dll` libs in the bundle. There is also XHarness.CLI tool with `mlaunch` inside to deploy `*.app` and `*.ipa` to a target (device or simulator) and listens for logs via sockets.
```

Reply:
btw. I don't think it makes sense to mention `mlaunch` here if we don't explain what it is, just creates confusion.